tck/faces23/localized-composite/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2022 Contributors to the Eclipse Foundation.
+    Copyright (c) 2017, 2021 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<faces-config xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_3.xsd"
+              version="2.3">
+    <application>
+        <locale-config>
+            <default-locale>en</default-locale>
+            <supported-locale>es</supported-locale>
+            <supported-locale>pt</supported-locale>
+            <supported-locale>pt_BR</supported-locale>
+            <supported-locale>pt_BR_PB</supported-locale>
+        </locale-config>
+        <resource-bundle>
+            <base-name>locale.messages</base-name>
```

Comment:
Should this be messages.properties instead?

Reply:
No, it shouldn't. This idiom is used to point to the file locale/messages.properties that will be placed by maven inside the classes directory, making it available to faces as locale.messages.

Reply:
Due to @brideck comment, I've built it again, and the locale/messages.properties is copied into the classes/ folder and, into the war, into the corresponding WEB-INF/classes/ folder. I've also merged upstream master into the pulling branch using the sync tool of github, but I'm not sure if it went smoothly (I should have use the plain old git to do that). If there is something I can do to help, do not hesitate to ask.

![image](https://user-images.githubusercontent.com/1236873/230179037-40fbfc8a-3bf6-4742-8f54-910a5bce3bbe.png)

Reply:
Thanks for checking. I can at least confirm that the test as written passes for me when I force those files into the app as shown above. I must be missing something in my env that would be making that happen automatically during app creation for me, although I can't identify what that might be.